includes/class-wc-payments-checkout.php

```diff
@@ -435,19 +443,19 @@ function () use ( $prepared_customer_data ) {
 									'number' => '<button type="button" class="js-woopayments-copy-test-number" aria-label="' . esc_attr( __( 'Click to copy the test number to clipboard', 'woocommerce-payments' ) ) . '" title="' . esc_attr( __( 'Copy to clipboard', 'woocommerce-payments' ) ) . '"><i></i><span>',
 								]
 							);
-				?>
+					?>
 				</p>
-				<?php
+					<?php
 			endif;
 
-			if ( $display_tokenization ) {
-				$this->gateway->tokenization_script();
-				// avoid showing saved payment methods on my-accounts add payment method page.
-				if ( ! is_add_payment_method_page() ) {
-					$this->gateway->saved_payment_methods();
+				if ( $display_tokenization ) {
+					$this->gateway->tokenization_script();
+					// avoid showing saved payment methods on my-accounts add payment method page.
+					if ( ! is_add_payment_method_page() ) {
+						$this->gateway->saved_payment_methods();
+					}
 				}
-			}
-			?>
+				?>
 
 			<fieldset style="padding: 7px" id="wc-<?php echo esc_attr( $this->gateway->id ); ?>-upe-form" class="wc-upe-form wc-payment-form">
```

Comment:
I find `wc-upe-form` and `wcpay-upe-form` named closely similar :D 

Since `wc-upe-form` is not used anywhere, could we remove it?

I wonder if we can just remove the `id` as well.